plugins/org.python.pydev.parser/src/org/python/pydev/parser/grammar_fstrings/build.xml

```diff
@@ -1,7 +1,7 @@
 <project name="jython-parser" default="parser">
 
-    <property name="javaccHome" value="c:\bin\javacc-5.0" />
-    <property name="python" value="c:\bin\Miniconda\envs\py27_tests\python.exe" />
+    <property name="javaccHome" value="C:\Users\olccjr\Documents\Javacc\javacc-5.0" />
```

Comment:
I think that we could use env variables for that (so, please update the build to use environment variables instead of hardcoding -- please add comments to say what are the expected values for the environment variables for other users).